logging/inc/leatherman/logging/logging.hpp

```diff
@@ -162,6 +162,13 @@ namespace leatherman { namespace logging {
      */
     void setup_eventlog_logging(std::string application);
 
+    /**
+     * Configures application to log to syslog
+     * The logging level is set to warning by default.
+     * @param application Destination stream for logging output.
+     */
+    void setup_syslog_logging(std::string application, int facility = (21<<3));
```

Comment:
should probably have a define for `(21<<3)`

Reply:
I'm not sure if this parameter should be optional at all

Reply:
makes sense